pkg/clusters/addons/kong/kong.go

```diff
@@ -112,7 +112,7 @@ func (a *Addon) Deploy(ctx context.Context, cluster clusters.Cluster) error {
 
 	// ensure the repo is added
```

Comment:
```suggestion
	// ensure the repo exists
```

small semantic difference - if we want the operation to be idempotent, we care about presence, not about addition.